access/src/main/webapp/WEB-INF/access-fedora-context.xml

```diff
@@ -96,7 +95,7 @@
 		<property name="externalContentPath" value="${external.base.url}" />
 	</bean>
 	
-	<bean class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
+<!--  	<bean class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
```

Comment:
delete this block?

Reply:
I think this is still going to be used, just appears to be commented out so things will start.  I'm not sure why this would fail though

Reply:
Seems to work fine. Maybe commented out because the referencing beans didn't see to be in the file